documentation/testing/test-strategy.md

```diff
@@ -89,54 +92,67 @@ be requested as appropriate.
 
 ##### Technology Selected
 
-** *add technology here* **
+[Xunit](https://github.com/xunit)
 
 #### Integration Testing
 
 Integration tests are used to ensure that parts of the code, and frameworks are work together in the correct way to create a working
 service. These work by compiling the code locally using mocked data on which actions and assertions are executed. Integration tests build upon
 the quality that is assured during unit testing but are faster and lighter than end-to-end tests allowing them to be run frequently, and earlier in CI.
-
+These tests will run in the build stage after code changes are made.
 These tests are the responsibility of the developers to write and maintain in collaboration with the test engineers.
 
 
 ##### Technology Selected
 
-** *add technology here* **
+[Moq](https://learn.microsoft.com/en-us/shows/visual-studio-toolbox/unit-testing-moq-framework)
+
+[Xunit](https://github.com/xunit)
 
 #### Feature/E2E Testing
 
 Feature and end-to-end tests are both designed to simulate the behaviour of an end user. Tests are written from an end-user perspective and run
 against deployed environments through a web-browser (such as Chrome, Safari, Firefox etc.) driven by web-drivers. This is the slowest type of (functional) automated
 testing are run later than unit and integration tests (see testing pyramid).
 
+These tests will run in testing environment after build stage in Testing environment. The testing environment will have pre loaded data to be used by tests. 
```

Comment:
Can you remind me where this testing environment is if we are using trunk based development? Do we have a separate testing environment set up, or is the current stage of the `main` branch essentially the testing enviroment?

Reply:
That's the decision we need to make to setup a separate environment for testing. I have marked this PR as draft until we have concluded what we are going to do.